umbraco/Umbraco-CMS
# Fix: Load DLLs from Package ZIPs

`Assembly.Load()` wont load any package ZIP DLLs that were not published as part of the app, it would be OK if it was a Nuget package reference to the website

## Reference
https://github.com/dotnet/runtime/issues/13511#issuecomment-537475896

## Test Notes
* Try uploading the ZIP package before PR applied to site in the backoffice
*  Note installing a package via the backoffice drag & drop causes the site to restart & NOT a fun user experience
* Site will YSOD/error with FileNotFound Exception
* Checkout PR and see if site will now boot & run

## Example ZIP
[Take_Out_The_Trash_2.0.0.zip](https://github.com/umbraco/Umbraco-CMS/files/6384976/Take_Out_The_Trash_2.0.0.zip)

@bergmania or @Shazwazza you able to take a look at this and ensure using `LoadFrom` not gonna be a pain or a security problem etc especially as this is to do with our own TypeLoader stuff
